packages/teams-js/src/public/runtime.ts

```diff
@@ -35,10 +35,11 @@ export interface IRuntime {
     readonly notifications?: {};
     readonly pages?: {
       readonly appButton?: {};
-      readonly tabs?: {};
-      readonly config?: {};
       readonly backStack?: {};
+      readonly config?: {};
       readonly fullTrust?: {};
+      readonly currentApp?: {};
```

Comment:
Is this supposed to be alphabetized? If so currentApp should be ahead of fullTrust

Reply:
Resolved in the latest commit.